spec.bs

```diff
@@ -1342,14 +1342,18 @@ two different transactions that a user U performs with payment instruments P1
 and P2 (either on the same merchant M, or two colluding merchants M1 and M2),
 the merchant(s) may now be able to learn that P1 and P2 are for the same user.
 
-For many current online payment flows this may not be a significant concern, as
-the user already provides sufficient information to do this joining anyway
-(e.g. their address), however it could become a privacy attack if, e.g.,
-payment tokenization becomes commonplace.
-
-One way a user may mitigate this concern is to use distinct user accounts for
-each payment instrument. It is not, however, guaranteed that a [=Relying
-Party=] (e.g., a bank) will not join those accounts internally.
+For many current online payment flows this may not be a significant
+risk, as the user often provides sufficient information to do
+this joining anyway (e.g., name, email address, shipping address).
+
+However, if payment methods that involve less identifying information
+(e.g., tokenization) become commonplace, it is important that
+ecosystem stakeholders take non-technical steps to preserve user
```

Comment:
Hi @npdoty. The simplest fix might be to remove "non-technical." Having said that, I think of bullets 2 and 3 as non-technical because "not-sharing" seems like a policy decision and "one credential per instrument" seems also like a policy constraint. I would be comfortable deleting "non-technical".

Reply:
Yeah, we're often vague about what qualifies as technical vs non-technical. Separating identifiers or minimizing data shared seem like technical mitigations to me, but I don't feel strongly about it.